chatbot/db/db.js

```diff
@@ -11,7 +11,19 @@ pool.on('error', (err) => {
     console.error('unexpected database error:', err)
 })
 
-function createSessionFromRow(r) {
+async function createSessionFromRow(client, r) {
+
+    if(!r) {
+        return null
```

Comment:
Is there any valid case where this would be invoked with a null row? If not, recommend throwing an error as this graceful handling will make it much harder to detect and understand a problem higher-up the chain.